app/helpers/remote_execution_helper.rb

```diff
@@ -230,6 +237,26 @@ def load_template_from_task(template_invocation, target)
     task.execution_plan.actions[1].try(:input).try(:[], 'script')
   end
 
+  def jobs_template_with_input(job_invocation)
```

Comment:
Could we split this method into two methods and return just one value?
It would be much more readable IMHO.
```suggestion
  def job_report_template
    ReportTemplate.where(name: Setting['remote_execution_job_invocation_template']).first
    return template if template && template.template_inputs.where(name: 'job_id').exists?
  end
  
  def job_report_template_parameters(job_invocation, template)
    template_input = template.template_inputs.where(name: 'job_id').first
    raise "#job_report_template_parameters need template that has 'job_id' input" unless template_input
    {
      report_template_report: {
        input_values: {
          "#{template_input.id}": {
            value: job_invocation.id,
          },
        },
      },
    }
  end
```